roles/control_node/tasks/30_controller.yml

```diff
@@ -96,6 +96,15 @@
     - [ "awx", "{{ username }}" ]
     - [ "quay.io/cbrandt/security_workshop_ee:latest", "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0", "registry.redhat.io/ansible-automation-platform-20-early-access/ee-29-rhel8:2.0.0", "registry.redhat.io/ansible-automation-platform-20-early-access/ee-minimal-rhel8:2.0.0" ]
 
+- name: Pull an extra image for f5
+  become_user: "{{ item[0] }}"
+  containers.podman.podman_image:
+    name: "{{ item[1] }}"
+  with_nested:
+    - [ "awx", "{{ username }}" ]
+    - ["quay.io/acme_corp/f5_modules:latest"]
+  when: workshop_type == "f5"
```

Comment:
@anshulbehl I would move this task to the `roles/control_node/tasks/workshops/f5.yml` so that you don't need a conditional.  Each `workshop_type` always has a YAML file so we can avoid conditionals and make it easier to read.

Reply:
+1

Reply:
@IPvSean @anshulbehl 
On line 97 of the file. I've just added the security ee to the existing list @IPvSean created.. My thoughts were to create a var "{{ security_ee_image }}" or a a "{{ ee_image_list}}" with all the images that need to be pulled for controller, and add defaults to the role. Open to comment as doing per workshop could save some time
```
[ "quay.io/cbrandt/security_workshop_ee:latest", "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0", "registry.redhat.io/ansible-automation-platform-20-early-access/ee-29-rhel8:2.0.0", "registry.redhat.io/ansible-automation-platform-20-early-access/ee-minimal-rhel8:2.0.0" ] 
```

Reply:
I can then add f5 ee to the list too

Reply:
why not just add the EEs to the specific workshop type?
roles/control_node/tasks/workshops/f5.yml
and 
roles/control_node/tasks/workshops/security.yml

I can actually move the current EE sync to the RHEL/security workshop, etc so that it is not installing it for the others to speed up provisioning?